op-node/Dockerfile

```diff
@@ -6,6 +6,8 @@ RUN apk add --no-cache make gcc musl-dev linux-headers git jq bash
 
 # build op-node with the shared go.mod & go.sum files
 COPY ./op-node /app/op-node
+COPY ./op-batcher /app/op-batcher
+COPY ./op-proposer /app/op-proposer
```

Comment:
I do not believe these should be dependencies of the op-node, since the op-batcher and op-proposer already rely on the op-node. The op-batcher and op-proposer codebases are not meant to leak into user land, even if it's only metrics.